manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -425,9 +425,27 @@ LimitRequestFieldSize 524288
   ProxyPreserveHost on
   <Location /assets/>
     Header unset ETag
+    Header set Content-Security-Policy            "default-src 'self'; child-src 'self'; connect-src 'self'; font-src 'self' fonts.gstatic.com; script-src 'unsafe-eval' 'unsafe-inline' 'self'; style-src 'unsafe-inline' 'self'; report-uri /dashboard/csp_report"
```

Comment:
A lot of these don't make sense...for example, I don't think we need `font-src fonts.gstatic.com` on the static assets, but maybe I'm wrong.  Same for things like unsafe-eval and unsafe-inline.

Reply:
I was avoiding picking and choosing. And went the blind approach.

I understand how these make sense from `.js`, but they don't make sense to me on `.jpg`. Unfortunately, I think that is what was requested.

Reply:
ok. I used the header that is set in `secure_headers.rb` for the values that I set into apache. Specifically, this is the C-S-P from `/ops/explorer`.

I had originally thought we could setup one C-S-P and put it into apache, removing it from rails all together.
My second thought was to have one C-S-P and duplicate it across `secure_headers.rb`, apache, and pods. There are a few controllers that tweak the policy as needed (i.e.: consoles and websockets).

Please confirm: You want our dynamic C-S-P set in `secure_headers.rb`
and a different static C-S-P set in apache/pods (synchronized)?

We do have an outstanding request to change the policy we use, but we stated that our login requires our current policy.